pkg/api/v1beta3/types.go

```diff
@@ -521,7 +521,9 @@ type Container struct {
 	// Required.
 	Image string `json:"image" description:"Docker image name"`
 	// Optional: Defaults to whatever is defined in the image.
-	Command []string `json:"command,omitempty" description:"command argv array; not executed within a shell; defaults to entrypoint or command in the image; cannot be updated"`
+	Command []string `json:"command,omitempty" description:"entrypoint array; not executed within a shell; defaults to the image's entrypoint; cannot be updated"`
+	// Optional: Defaults to the image's entrypoint
```

Comment:
Terminology nit (here and elsewhere): The term "default" everywhere else implies that a GET on the object will include the default value. That would require communicating the value up from Kubelet, which we could do, but currently don't, last I looked. I would just say the image's entrypoint is used if a value isn't provided.